src/toolkits/object_detection/object_detector.cpp

```diff
@@ -353,23 +353,22 @@ void object_detector::load_version(iarchive& iarc, size_t version) {
 void object_detector::import_from_custom_model(variant_map_type model_data,
                                                size_t version) {
   auto it = model_data.find("_model");
-  flex_dict model = variant_get_value<flex_dict>(it->second);
-  model_data.erase(it);
+  const flex_dict& model = variant_get_value<flex_dict>(it->second);
   auto it2 = model_data.find("_grid_shape");
```

Comment:
Since `it` and `it2` are used farther down in the function, could we give them more descriptive names?